Library/ViewModels/PledgePaymentMethodsViewModel.swift

```diff
@@ -289,6 +289,13 @@ public final class PledgePaymentMethodsViewModel: PledgePaymentMethodsViewModelT
     self.shouldCancelPaymentSheetAppearance <~ updatedCards.signal
       .mapConst(true)
 
+    // Only ever use the value if the view model is configured and the payment sheet could exist.
+    let safeShouldCancelPaymentSheet = Signal.combineLatest(
+      self.shouldCancelPaymentSheetAppearance.signal,
+      configureWithValue
+    )
+    .map(first)
```

Comment:
Nit: Most patterns I've seen in our codebase call `configureWithValue` very early, like in `onAppear`. When does this issue happen? Just seems a bit odd to need the protection.

Reply:
It happens in the logged out state! If the user is logged out, the payment sheet doesn't actually get configured, but we still say "hey, please don't show that extra modal" if the user interacts with the pledge cta (ex: if they want to log in). Leaving it as it was causes the app to crash when that happens.

Reply:
Aha, that makes sense! This would be a good comment, actually.